solutions/CSES Sliding Median.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: cses-1076
+source: CSES
+title: Sliding Median
+author: Isaac Noel
+---
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
+
+# Abstract
```

Comment:
Use second-tier headings (## instead of #) to be consistent with how other solutions are formatted.